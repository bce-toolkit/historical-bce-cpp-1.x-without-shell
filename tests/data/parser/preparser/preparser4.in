F2+I2=IF7