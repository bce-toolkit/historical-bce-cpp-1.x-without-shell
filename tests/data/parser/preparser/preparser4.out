F2 0 0
I2 0 0
IF7 1 0
